### PR TITLE
Feature #3846 - Generalizing component intro

### PR DESCRIPTION
**Pull request by NicolasEYSSERIC**
Do not forget to check out PR on Silverpeas-Components...

### CODE REVIEW

File: core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ */
+package org.silverpeas.core.web.util.viewgenerator.html.instanceintro;
+
+import org.apache.ecs.html.Div;
+import org.silverpeas.core.admin.component.model.ComponentInstLight;
+import org.silverpeas.core.admin.service.OrganizationController;
+import org.silverpeas.core.contribution.content.wysiwyg.service.WysiwygController;
+import org.silverpeas.core.util.SettingBundle;
+import org.silverpeas.core.util.StringUtil;
+import org.silverpeas.core.web.util.viewgenerator.html.GraphicElementFactory;
```
Comment (silveruser) [inline core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java]: MINOR Remove this unused import 'org.silverpeas.core.web.util.viewgenerator.html.GraphicElementFactory'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

File: core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import org.silverpeas.core.web.util.viewgenerator.html.wysiwyg.WysiwygEditor;
```
Comment (silveruser) [inline core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java]: MINOR Remove this unused import 'org.silverpeas.core.web.util.viewgenerator.html.wysiwyg.WysiwygEditor'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

File: core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+import javax.servlet.http.HttpSession;
```
Comment (silveruser) [inline core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java]: MINOR Remove this unused import 'javax.servlet.http.HttpSession'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

File: core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.silverpeas.core.web.util.viewgenerator.html.instanceintro;
+
+import org.apache.ecs.html.Div;
```
Comment (silveruser) [inline core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java]: MINOR Remove this unused import 'org.apache.ecs.html.Div'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

File: core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+import org.silverpeas.core.admin.component.model.ComponentInstLight;
+import org.silverpeas.core.admin.service.OrganizationController;
+import org.silverpeas.core.contribution.content.wysiwyg.service.WysiwygController;
+import org.silverpeas.core.util.SettingBundle;
```
Comment (silveruser) [inline core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/instanceintro/ComponentInstanceIntroTag.java]: MINOR Remove this unused import 'org.silverpeas.core.util.SettingBundle'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

File: core-web/src/main/java/org/silverpeas/core/web/mvc/route/ComponentRequestRouter.java [hunk 6]
```diff
@@ -23,9 +23,14 @@
  */
 package org.silverpeas.core.web.mvc.route;
 
+import org.silverpeas.core.admin.component.model.ComponentInst;
```
Comment (silveruser) [inline core-web/src/main/java/org/silverpeas/core/web/mvc/route/ComponentRequestRouter.java]: MINOR Remove this unused import 'org.silverpeas.core.admin.component.model.ComponentInst'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SilverYoCha** [comment]
Merged.
Closed manually because of manual git operations performed during integration.

**Pull request closed without merging**